integration/airflow/tests/integration/tests/airflow/dags/secrets.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+from airflow.models import BaseOperator
+from airflow.utils.dates import days_ago
+
+from openlineage.client import set_producer
+
+set_producer("https://github.com/OpenLineage/OpenLineage/tree/0.0.1/integration/airflow")
+
+from airflow.version import version as AIRFLOW_VERSION
+from pkg_resources import parse_version
+if parse_version(AIRFLOW_VERSION) < parse_version("2.0.0"):
+    from openlineage.airflow import DAG
+else:
+    from airflow import DAG
+
+
+"""This shouldn't have extractor - we're testing if we'll see password in UnknownSourceAttribute"""
+class SecretsOperator(BaseOperator):
+    template_fields = ['password']
+
+    def __init__(self, password, **kwargs):
+        super().__init__(**kwargs)
+        self.password = password
+
+    def execute(self, context):
+        logging.getLogger(__name__).warning(f"Password! {self.password}")
```

Comment:
This brings me to thought it could be also unit tested with logs. See how SecretsMasker is tested in Airflow as an example with `caplog` fixture [here](https://github.com/apache/airflow/blob/main/tests/utils/log/test_secrets_masker.py)